resources/migrations/000_migrations.yaml

```diff
@@ -8495,6 +8495,36 @@ databaseChangeLog:
               )
               WHERE dashboard_id IS NOT NULL;
 
+  - changeSet:
+      id: 314
+      author: howonlee
+      comment: Added 0.41.0 Fine grained caching controls
+      changes:
+        - addColumn:
+            tableName: metabase_database
+            columns:
+              - column:
+                  name: cache_ttl
+                  type: integer
+                  remarks: "Granular cache TTL for specific database."
+                  constraints:
+                    nullable: true
+
+  - changeSet:
```

Comment:
Why two different changeSets here?

Reply:
relaying cam's reason why: mysql can get to a terrible awful no good state if it adds first column, borks, and then tries again because it's not reentrant in that way and there's not a txn wrapping it.